client/layout/logged-out.jsx

```diff
@@ -16,6 +16,7 @@ import wooDnaConfig from 'calypso/jetpack-connect/woo-dna-config';
 import MasterbarLoggedOut from 'calypso/layout/masterbar/logged-out';
 import OauthClientMasterbar from 'calypso/layout/masterbar/oauth-client';
 import isA8CForAgencies from 'calypso/lib/a8c-for-agencies/is-a8c-for-agencies';
+import isAkismetRedirect from 'calypso/lib/akismet/is-redirect-from-akismet';
```

Comment:
To be consistent with other similar helpers, I think the function naming should match the filename here.

So either call the function `isRedirectFromAkismet` or rename the file `is-akismet-redirect`.

Reply:
Fixed in [3300d9c](https://github.com/Automattic/wp-calypso/pull/100973/commits/3300d9c34f52dc5e2c34a8c7c70ddab9cb5488d8)